yarn-project/aztec.js/src/fee/private_fee_payment_method.ts

```diff
@@ -68,7 +68,7 @@ export class PrivateFeePaymentMethod implements FeePaymentMethod {
       ),
       to: this.asset,
     });
-    await this.wallet.createAuthWitness(messageHash);
+    await this.wallet.createAuthWit(messageHash);
```

Comment:
Here is the incosistency. The message in  createAuthWit(...) is messageHash here.